### ISSUE TITLE
Cancelling doesn't actually cancel.

### ISSUE THREAD

**Issue by micahlisonbee** [moxley/atom-ruby-test#95]
`control + cmd + c` prints "Tests canceled" in the console. It also stops the console from displaying any more tests. However, I don't think it actually cancels the tests, they seem to be running in the background still.

What brought this to my attention is Firefox keeps popping open for Capybara tests at the end of my  suite well after I send the kill command.

Also, emails generated from my tests were still going out.

**Comment by moxley** [comment]
Yeah, interesting. I've always wondered if cancelling the tests actually killed the process or if it just make it silent. Looks like you have a way to determine that. I don't know what the solution would be.
